nextflow.config

```diff
@@ -216,6 +217,12 @@ profiles {
         executor.cpus           = 4
         executor.memory         = 8.GB
     }
+    gpu {
```

Comment:
If the pipeline is started with this profile, the specified settings will be applied to every process.

I think we should apply these settings only to `PARABRICKS_FQ2BAMMETH`. Or am I missing something?